.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+### Type of this PR 
+
+- [ ] Bug fix
+- [ ] Improvement
+- [ ] Documentation
+- [ ] Cleanup
+- [ ] Test
+
+### What does this PR do?
+
+...
+
+### Screenshot/screencast of this PR
+
+...
+ 
+
+### Potential side effects
+
+...
+
+### How to test this PR?
+
+- [ ] JUnit
```

Comment:
More like "Unit tests" in general. JUnit for the backend parts, but jest for the front.

Reply:
+ I've split in
- [ ] Frontend Unit tests
- [ ] Backend Unit tests